src/uuid.ts

```diff
@@ -0,0 +1,9 @@
+import { v4 } from "uuid";
+
+export default class UUID {
+  readonly value: string;
+
+  constructor() {
+    this.value = v4();
```

Comment:
is there a reason `v4()` is not used directly in https://github.com/phrase/i18next-phrase-backend/pull/31/files#diff-aafd2a619aa1cb5c9ba3bae3b27169458a297d19abba55c25cc8a74b398b3c63R103 ?
not sure if adding another file just as a thin wrapper makes sense, unless it brings some non-obvious benefits.

Reply:
Well not much sense, I agree. I had plans in near future to add some logic to the uuid class:

- if there is `crypto` API available in global scope, use the `randomUUID` method for UUID generation
- otherwise dynamically load the `uuid` lib and use the `v4` method for UUID

That way we wouldn't load the unnecessary lib in browser since there is already `crypto` API. I will remove the UUID class for now since it is not useful ATM.